source/val/validate_annotation.cpp

```diff
@@ -163,6 +163,7 @@ spv_result_t ValidateDecorationTarget(ValidationState_t& _, spv::Decoration dec,
     case spv::Decoration::Stream:
     case spv::Decoration::RestrictPointer:
     case spv::Decoration::AliasedPointer:
+    case spv::Decoration::PerPrimitiveNV:
```

Comment:
```suggestion
    case spv::Decoration::PerPrimitiveEXT: // alias PerPrimitiveNV
```

(makes it easier to grep/code search for the EXT name in the future)

Reply:
Done